packages/schemas/src/types/subject-token.ts

```diff
@@ -0,0 +1,11 @@
+import { jsonObjectGuard } from '@logto/connector-kit';
+import { number, object, string, type z } from 'zod';
+
+export const subjectTokenResponseGuard = object({
+  subjectToken: string(),
+  expiresIn: number(),
+  userId: string(),
+  context: jsonObjectGuard,
+});
```

Comment:
Not aligned with [efc](https://github.com/logto-io/rfcs/blob/master/draft/0005-impersonation.md#response-1)?

Reply:
updated

Reply:
what is efc? @simeng-li